docs/user/aws.rst

```diff
@@ -0,0 +1,103 @@
+=======================================
+AWS Bootstrapping using Geodocker Guide
+=======================================
+Getting started on AWS is incredibly simple thanks to the `Geodocker <https://github.com/geodocker/geodocker-accumulo-geomesa>`_ project.  The following guide describes how to bootstrap a GeoMesa Accumulo cluster using Amazon ElasticMapReduce and Docker and ingesting and querying some sample data.  This guide assumes you have an Amazon Web Services account already provisioned as well as an IAM key pair.  To set up the AWS command line tools, follow the instructions found in the AWS `online documentation <http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html>`_.
+
+Bootstrap an EMR cluster
+************************
+Use the following command to bootstrap an EMR cluster.  You will need to change the KeyName to the IAM key pair you intend to use for this cluster.  You can also edit the instance types to a size appropriate for your use case.  Make sure you adjust the Accumulo cache configuration settings accordingly.  For instance, if you use a high memory instance type, you will want to increase the `TSERVER_XMX` parameter which controls the amount of heap space allocated to the JVM running the Accumulo Tablet Server.  Additionally, you should adjust the `TSERVER_CACHE_DATA_SIZE` and the `TSERVER_CACHE_INDEX_SIZE` to appropriate fractions of the `TSERVER_XMX` memory to take advantage of the increase in memory.
+
+.. code-block:: shell
+
+  aws emr create-cluster                                \
+      --name "GeoDocker GeoMesa"                        \
+      --release-label emr-5.2.0                         \
+      --output text                                     \
+      --use-default-roles                               \
+      --ec2-attributes KeyName=__KEY_NAME__             \
+      --applications Name=Hadoop Name=Zookeeper Name=Spark \
+      --instance-groups                                    \
+        Name=Master,InstanceCount=1,InstanceGroupType=MASTER,InstanceType=m3.xlarge \
+        Name=Workers,InstanceCount=3,InstanceGroupType=CORE,InstanceType=m3.xlarge  \
+      --bootstrap-actions                                                                        \
+        Name=BootstrapGeoMesa,Path=s3://geomesa-docker/bootstrap-geodocker-accumulo.sh,Args=\[-i=quay.io/geomesa/accumulo-geomesa:latest,-n=gis,-p=secret,-e=TSERVER_XMX=10G,-e=TSERVER_CACHE_DATA_SIZE=6G,-e=TSERVER_CACHE_INDEX_SIZE=2G\]
+
+
+After executing that command, you can monitor the state of the EMR bootstrap process
+by going to the `Management Console <https://console.aws.amazon.com/elasticmapreduce/home?region=us-east-1#cluster-list>`_.  Find the name (as specified in the aws emr command) of the cluster and click through to its details page.  Under the 'Hardware' section, you can find the master node and it's IP address.  Copy the IP address and then run the following command.
```

Comment:
"it's IP address" -> "its IP address". Also, might want to put aws emr in double backticks (see above).